src/Network/Network/VirtualRouter/GetAzureRMVirtualRouterCommand.cs

```diff
@@ -31,7 +31,7 @@ public partial class GetAzureRmVirtualRouter : VirtualRouterBaseCmdlet
     {
         [Parameter(
             ParameterSetName = VirtualRouterParameterSetNames.ByVirtualRouterName,
-            Mandatory = true,
+            Mandatory = false,
```

Comment:
mandatory parameter cannot be optional in non-breaking change release, please add another parameter set for subscription level list operation